crates/turborepo-lib/src/run/cache.rs

```diff
@@ -360,6 +364,21 @@ impl TaskCache {
             globwalk::WalkType::All,
         )?;
 
+        debug!("files to cache: {:?}", files_to_be_cached.len());
```

Comment:
Maybe add in the task id here
```suggestion
        debug!("{}: files to cache: {:?}", self.task_id, files_to_be_cached.len());
```